src/libraries/System.Private.CoreLib/src/System/IO/FileSystem.Unix.cs

```diff
@@ -33,6 +33,13 @@ public static void CopyFile(string sourceFullPath, string destFullPath, bool ove
             long fileLength;
             UnixFileMode filePermissions;
             using SafeFileHandle src = SafeFileHandle.OpenReadOnly(sourceFullPath, FileOptions.None, out fileLength, out filePermissions);
+
+            // Try to clone the file first.
+            if (TryCloneFile(sourceFullPath, destFullPath, overwrite, CreateOpenException))
```

Comment:
Rather than passing `CreateOpenException` as an argument, can you move it out of the method and rename it to `CreateOpenExceptionForCopyFile`?